### DISCUSSION TITLE
Preview skeleton before anchor

### DISCUSSION THREAD

**Discussion by Efor18** [sceneview/sceneform-android#95] [Q&A]
Hi, I want to show a transparent preview of a model before anchor it into the plane and render it properly. Is there a way of doing this with sceneform? Can anyone help me or point me in the right direction?

See this [Image](https://ibb.co/jr0Q62h) as an example of what I want to achieve.

Thanks in advance!

**Comment by ThomasGorisse** [comment]
You will have to change the material color like this:
```
MaterialFactory.makeTransparentWithColor(this, Color(1.0f, 1.0f, 1.0f, 0.25f)
    .thenAccept { material ->
       renderable.setMaterial(material)
    })
```

**Reply by Efor18** [reply]
Thanks for your reply @ThomasGorisse, But what I want to achieve is what is shown in this [video](https://files.fm/u/g88wesk6d). Show a kind of preview of where the object will be place before is placed (anchored). Do you know if there is a way of doing this with Sceneform?

Thanks in advance!

**Reply by ThomasGorisse** [reply]
There is a need to update the actual code and make the PlaneVisualizer.planeRenderable accessible.

Can you make a PR for "PlaneRender custom renderable"?

**Reply by ThomasGorisse** [reply]
Or simply place the model on the scene center. It will follow the camera without any link with ARCore:
```
Node modelNode = new Node();
modelNode.setRenderable(model);
modelNode.setLocalScale(new Vector3(0.3f, 0.3f, 0.3f));
modelNode.setLocalRotation(Quaternion.axisAngle(new Vector3(0f, 1f, 0f), 180));
modelNode.setLocalPosition(new Vector3(0f, 0f, -1.0f));
arSceneView.getScene().addChild(modelNode2);
```

**Reply by Efor18** [reply]
Thank you very much!
